### DISCUSSION TITLE
FlinkSQL 操作mysql-cdc 到Doris的数据同步过程中，由于mysql库中字段有换行符，导致数据Load失败

### DISCUSSION THREAD

**Discussion by Qichen-Kylin** [apache/doris#15064] [A - General / Q&A]
FlinkSQL 操作mysql-cdc 到Doris的数据同步过程中，由于mysql库中字段有换行符，导致数据Load失败，请问如何解决？

**Comment by LemonLiTree** [comment]
可以自己写demo source读取完在flink 清洗一下，再sink到doris

**Comment by zy-kkk** [comment]
可以在sink的配置项添加`'sink.properties.column_separator' = '&'`和`'sink.properties.line_delimiter'='$'`等比较复杂的分隔符和换行符
